packages/material-ui/src/SvgIcon/SvgIcon.d.ts

```diff
@@ -15,7 +15,7 @@ export interface SvgIconTypeMap<P = {}, D extends React.ElementType = 'svg'> {
     /**
      * The fontSize applied to the icon. Defaults to 24px, but can be configure to inherit font size.
      */
-    fontSize?: 'inherit' | 'default' | 'small' | 'large';
+    fontSize?: 'inherit' | 'large' | 'medium' | 'small';
```

Comment:
If this is a deprecation, shouldn't we have both `default` and `medium` to allow migration? Otherwise, we are breaking the build with this change.